spec/datatables/user_datatable_spec.rb

```diff
@@ -95,55 +95,45 @@
     let(:output) { subject.as_json }
 
     it 'recordsTotal' do
-      skip('This fails since Ruby 2.5.8')
```

Comment:
This is reassuring! Somewhere in my fork with the ruby 2.6+ and data tables updates these have been passing for me too.